analytic_amount_security/readme/CONFIGURE.md

```diff
@@ -0,0 +1,14 @@
+By itself, the module respects the original permissions. All internal users will be able to access the amounts of the analytic lines to which they have access.
```

Comment:
issue: I don't understand this sentence... shouldn't it be like this?

```suggestion
Users will continue to access the the same analytic lines as before. They just won't see the cost field unless explicitly granted.
```

Reply:
The idea here is that the module keeps the standar behavior by default as we want to avoid integration issues as the one that raised in these tests. Then, bring or remove permissions at will in every customer instance.

Reply:
I think that's not the correct approach. If one wants to keep the same behavior, just don't install the module.

What I would expect is that, after installing this module, nobody has access to analytic costs. Then the sysadmin will have to grant that access specifically to those that require it (either by clicking their user boxes or altering groups inheritances).

If that breaks others' tests... well, it's expected because this module would then alter Odoo's default behavior. Then this module should be declared as rebel, just like in https://github.com/OCA/crm/pull/645.

Of course it's true that this has the potential to break existing flows, so I think it would be useful to explain that clearly here.

Reply:
> I think that's not the correct approach. If one wants to keep the same behavior, just don't install the module.

The issue here is that you're going to break legitimate users workflows if you remove those rights completely to begin with (e.g.: accounting)

In my opinion, a module that breaks stuff (as it's going to happen) when installed can't be correct. Maybe is enough to leave `analytic.group_analytic_accounting` anyway